deel-3-webpagina/index.html

```diff
@@ -7,10 +7,40 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
-        <h1>All Your Business Data in One Place</h1>
-        <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
-            analytics tools.</h3>
-    </div>
+<nav class="navbar">
+    <ul>
+        <li>ANALITICA</li>
```

Comment:
Zet nooit je html code in hoofdletters. Maar transformeer de tekst naar hoofdletters in je CSS.